### DISCUSSION TITLE
Zydis bindings and examples for Free Pascal

### DISCUSSION THREAD

**Discussion by 440bx** [zyantific/zydis#514] [Show and tell]
This is just to make it known that Pascal bindings for the Zydis dll and FPC versions of most (not all) examples are now available at https://forum.lazarus.freepascal.org/index.php/topic,67665.0.html

Hopefully those bindings and examples will be useful to others.

**Comment by ghost** [comment]
Have you uploaded ZydisInfoC.7z? I couldn't seem to find the archive.

**Comment by 440bx** [comment]
There is no ZydisInfoC.7z

Is such a file mentioned someplace ? and if it is, please point out where so I can make any necessary corrections.

CLARIFICATION: there is no ZydisInfoC.7z but, there is a ZydisInfoC in the folder 300_ZydisInfoC which is in the ZydisE.7z archive.

IOW, expand ZydisE.7z and in the directory 300_ZydisInfoC you'll find ZydisInfoC.

HTH.
